catalog/glue/glue_test.go

```diff
@@ -20,6 +20,7 @@ package glue
 import (
 	"context"
 	"errors"
+	"github.com/apache/iceberg-go/table"
```

Comment:
Please use gofmt to reorder the imports. We should probably add a linter step to identify and fix this....

Reply:
Let me handle that linter step. I will do some time later this week

Reply:
Might be tmr if I can squeeze in some time to do it :pray: